dashboard/client/components/SubMenu/index.tsx

```diff
@@ -5,30 +5,35 @@ import React, { Fragment } from 'react'
 import './styles.scss'
 
 const SubMenu = ({ spaces, space }) => {
-  const x = (id, name) => _ => Router.push({ pathname: '/home', query: { space: id } })
+  const x = (id, name) => _ => Router.push({ pathname: '/tickets', query: { space: id } })
 
   return (
     <div className="sub-menu">
       <div className="sub-menu__header">Developer Relations Dashboard</div>
       <div className="sub-menu__wrapper">
-        <div className={'sub-menu__title' + (!space ? '-selected' : '')}>
+        <div className={'sub-menu__title' + (space ? '' : '-selected')}>
           <div>All Spaces</div>
           <div>378</div>
         </div>
         {spaces &&
-          spaces.map(({ id, name }) => (
-            <Fragment key={id}>
-              <div
-                onClick={x(id, name)}
-                className={'sub-menu__items' + (space === id ? '-selected' : '')}
-                key={id}
-              >
-                <div>{name}</div>
-                <div>{Math.round(Math.random() * 100)}</div>
-              </div>
-              <div className="test" />
-            </Fragment>
-          ))}
+          spaces.map(({ id, name, totalCount }) => {
+            if (name === 'Default' || name === 'Help') {
+              return null
+            }
+            return (
+              <Fragment key={id}>
+                <div
+                  onClick={x(id, name)}
+                  className={'sub-menu__items' + (space === id ? '-selected' : '')}
+                  key={id}
+                >
+                  <div>{name}</div>
+                  <div>{totalCount}</div>
```

Comment:
i think u put totalCount on question ?

Reply:
ya, i need to call `getTickets` on each space to calculate how many total unanswered questions there are in each space. i just didn't get around to it last night cos i fell asleep.

<img width="310" alt="Screen Shot 2019-06-26 at 10 50 55 AM" src="https://user-images.githubusercontent.com/2607929/60202886-64330d00-9800-11e9-8ef8-a18c96826025.png">

is there a better way to do that?

Reply:
yeah i can put it into the backend and jsut send it down w spaces